ddocs/medic-admin/rewrites.json

```diff
@@ -1,6 +1,7 @@
 [
   {"from": "/",             "to": "index.html"},
-  {"from": "/main.css",     "to": "main.css"},
+  {"from": "/css/*",        "to": "css/*"},
+  {"from": "/fonts/*",      "to": "fonts/*"},
```

Comment:
I think we should be consistent about all attachments and prefix the others well - just like Webapp.

Reply:
Ya that makes sense. Looks like only /js/ resources in admin, so I moved those.